playbooks/puppet-pulpcore.yml

```diff
@@ -0,0 +1,96 @@
+# This playbook uses https://github.com/theforeman/puppet-pulpcore to deploy a standalone pulp3 server.
+# It is useful both for testing changes to puppet-pulpcore as well as for setting up a pulp3 playground environment.
+#
+# You can edit the task "Clone puppet-pulpcore" to point to a different remote or branch as necessary.
+#
+# You can edit the task "Deploy test.pp" if you want to test the module using some different params.
+#
+# To use the playbook, first create a box, e.g. centos7: `$ vagrant up centos7`
+# Then run the playbook on the box you created: `$ ansible-playbook -l centos7 playbooks/puppet-pulpcore.yml`
+---
+- hosts: all
+  become: true
+  roles:
+    - puppet_repositories
+    - epel_repositories
+
+  tasks:
+    - name: "Enable pulpcore repository"
+      yum_repository:
+        name: pulpcore-nightly
+        description: Pulpcore nightly YUM repo
+        baseurl: https://fedorapeople.org/groups/katello/releases/yum/nightly/pulpcore/el7/x86_64/
```

Comment:
This limits this to EL7, with EL8 on the horizon you can make this pull the dist from Ansible facts to work on both.